src/Features/CSharp/Portable/Completion/CompletionProviders/NamedParameterCompletionProvider.cs

```diff
@@ -226,6 +227,25 @@ private static ISet<string> GetExistingNamedParameters(BaseArgumentListSyntax ar
             return null;
         }
 
+        private static IEnumerable<ImmutableArray<IParameterSymbol>> GetRecordBaseTypeParameterLists(
+            SemanticModel semanticModel,
+            int position,
+            PrimaryConstructorBaseTypeSyntax recordBaseType,
+            CancellationToken cancellationToken)
+        {
+            var within = semanticModel.GetEnclosingNamedTypeOrAssembly(position, cancellationToken);
+            if (within != null)
+            {
+                var constructors = semanticModel.GetSymbolInfo(recordBaseType, cancellationToken).CandidateSymbols.OfType<IMethodSymbol>();
```

Comment:
doesn't this grab all *methods*?  we should filter down to instance constructors right?

Reply:
what about .InstanceConstructors?